### PR TITLE
SQL-774: Use the MongoSQL dialect by default

### PR DESCRIPTION
**Pull request by rychipman**
As discussed in standup today, a small change to switch the default dialect from "mysql" to "mongosql"

### CODE REVIEW

File: src/main/java/com/mongodb/jdbc/MongoDriver.java [hunk 1]
```diff
@@ -154,7 +156,7 @@ private void reconcileProperties(DriverPropertyInfo[] driverPropertyInfo, Proper
     private MongoConnection createConnection(ConnectionString cs, Properties info)
             throws SQLException {
         // attempt to get DIALECT property, and default to "mysql" if none is present
```
Comment (nbagnard) [inline src/main/java/com/mongodb/jdbc/MongoDriver.java:158]: "mysql" -> "mongosql"

**Review by nbagnard — APPROVED** [review]
LGTM! There is just one "mysql" left in one of the comment. I flagged it in a comment.

File: src/main/java/com/mongodb/jdbc/MongoDriver.java (hunk 1 above)
Comment (rychipman) [reply src/main/java/com/mongodb/jdbc/MongoDriver.java:158]: Thank you for catching this!

**Comment by rychipman** [comment]
evergreen merge

**Pull request merged**